_posts/2018-11-26-resize-text.md

```diff
@@ -27,18 +27,26 @@ When someone resizes text, any of the following can break the page:
 * content is lost or truncated
 * interactive elements overlapping each other
 
+The importance is that *relative units* (_em_, _rem_, _%_, _vw_) are used for text.
+
 ## How to test
 
 Testing is pretty straight forward.
 
-1. Choose the right browser
+1. Initial test in multiple browsers
+1. Resize text
+1. Confirm relative font usage
+1. Choose a browser to do further testing in
 1. Resize text
 1. Check everything
 
 ### Choosing a browser
 
-Firefox is the best browser for testing [WCAG Resize Text (Success Criterion 1.4.4)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html).
+Testing text-resize needs initial testing in multiple browsers to confirm functionality - that relative units are used correctly in order to conform with [WCAG Resize Text (Success Criterion 1.4.4)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html).
+
+*Testing text-resizing can fail in _Chrome_ but pass in _Firefox_* since text-zoom in Firefox resizes regardless of usage of relative units. Suggested setup is to use Chrome and Firefox in initial testing.
```

Comment:
I don't think we need the italic treatment, Mind removing it?

Reply:
Removed